zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/UserTaskQueryController.java

```diff
@@ -29,6 +31,7 @@
 public class UserTaskQueryController {
 
   private final UserTaskServices userTaskServices;
+  @Autowired private FormServices formServices;
```

Comment:
I would inject the `FormServices` also in the constructor like in the other controllers.